src/components/App.jsx

```diff
@@ -1,16 +1,35 @@
 import React from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
-// eslint-disable-next-line no-unused-vars
 import Calculate from '../logic/calculate';
 
-function App() {
-  return (
-    <div className="app">
-      <Display />
-      <ButtonPanel />
-    </div>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(buttonName) {
+    // console.log(`This is the buttonName on App component: ${buttonName}`);
```

Comment:
Let's remove `console.log`
No `console.log` committed to the repo. It's ugly, it kills performance. and it can make confidential data be visible in the browser tools.

**I trust you can do this before merging**; 

**👀 on you 🙂**